lumigator/backend/backend/repositories/secrets.py

```diff
@@ -10,6 +10,18 @@ class SecretRepository(BaseRepository[SecretRecord]):
     def __init__(self, session: Session):
         super().__init__(SecretRecord, session)
 
+    def delete_secret(self, name: str) -> bool:
+        """Delete a secret identified by its name.
+
+        :param name: The name of the secret to be deleted
+        :returns: ``True`` if the secret was deleted, ``False`` if the secret was not found, or not deleted.
```

Comment:
Actually the only possibility is to have a deleted count of 0 that means that the secret was not found in the repo, right????

```suggestion
        :returns: ``True`` if the secret was deleted, ``False`` if the secret was not found.
```